libraries/classes/InsertEdit.php

```diff
@@ -1630,6 +1631,13 @@ public function getCurrentValueAsAnArrayForMultipleEdit(
                     . $this->dbi->escapeString($multiEditSalt[$key]) . "')";
             }
 
+            if (
+                $multiEditFuncs[$key] === 'NOW'
+                && (is_numeric($currentValue) && $currentValue >= 0 && $currentValue <= 6)
+            ) {
+                return $multiEditFuncs[$key] . '(' . $this->dbi->escapeString($currentValue) . ')';
+            }
+
```

Comment:
This will not allow `NOW('7')` or any number not between 0 and 6.

```suggestion
            if ($multiEditFuncs[$key] === 'NOW') {
                if ((is_numeric($currentValue) && $currentValue >= 0 && $currentValue <= 6)) {
                    return $multiEditFuncs[$key] . '(' . $this->dbi->escapeString($currentValue) . ')';
                }

                return $multiEditFuncs[$key] . '()';
            }

```

Reply:
Yeah. but wouldn't it be best to let the user know that the value is wrong? I think we should not change the behaviour, and pass the value to the query.

`INSERT INTO `test` (`id`, `ts`) VALUES (NULL, NOW('7'))`

with your change we will have

`INSERT INTO `test2` (`id`, `ts`) VALUES (NULL, NOW())`

Reply:
Maybe show a hint when choosing `NOW()` that the value should be between 0 and 6, and while writing the value don't allow to enter something other than numbers between 0 and 6.

Reply:
@faissaloux can you check it with last changes? I also added a second video to PR description.

Reply:
From the video it appears that it shows a wrong alert message `Please enter a value less than or equal to 0` when you entered 3.

And I think it's better to be `Please enter a value between 0 and 6` as from `less than or equal 6`   it appears that it allows negative numbers too.

Reply:
@liviuconcioiu When I enter a valid value on a timestamp without length, it shows a false error message.
![image](https://github.com/user-attachments/assets/219b631a-75ff-4997-8142-3ba8aa56ea7f)

This is due to `data-decimals` attribute which value is 0 on timestamps without length.

Reply:
Not sure about this, because even if you put 5 there, it won't store the precision to the timestamp. So the message should be correct. The only accepted value is empty for `NOW()` or 0 for `NOW(0)`.

Reply:
No, actually for `timestamp`, `NOW()` accepts any value between 0 and 6.

Reply:
Yeah, but isn't stored.

Reply:
Yes true, I got you now. https://github.com/phpmyadmin/phpmyadmin/pull/19648#issuecomment-2759971521